src/js/jsx/sections/export/LayerExports.jsx

```diff
@@ -0,0 +1,297 @@
+ * all copies or substantial portions of the Software.
+ *  
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+ * DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var React = require("react"),
+        Fluxxor = require("fluxxor"),
+        FluxMixin = Fluxxor.FluxMixin(React),
+        StoreWatchMixin = Fluxxor.StoreWatchMixin,
+        _ = require("lodash"),
+        Immutable = require("immutable");
+
+    var Gutter = require("jsx!js/jsx/shared/Gutter"),
+        Label = require("jsx!js/jsx/shared/Label"),
+        Button = require("jsx!js/jsx/shared/Button"),
+        SVGIcon = require("jsx!js/jsx/shared/SVGIcon"),
+        Datalist = require("jsx!js/jsx/shared/Datalist"),
+        TextInput = require("jsx!js/jsx/shared/TextInput");
+
+    var collection = require("js/util/collection");
+
+
+    var scaleOptions = Immutable.OrderedMap({
+        "0.5": {
+            id: "0.5",
+            title: "0.5x"
+        },
+        "1": {
+            id: "1",
+            title: "1x"
+        },
+        "1.5": {
+            id: "1.5",
+            title: "1.5x"
+        },
+        "2": {
+            id: "2",
+            title: "2x"
+        }
+    });
+
+    var formatOptions = Immutable.OrderedMap({
+        "png": {
+            id: "png",
+            title: "PNG"
+        },
+        "jpg": {
+            id: "jpg",
+            title: "JPG"
+        },
+        "svg": {
+            id: "svg",
+            title: "SVG"
+        },
+        "pdf": {
+            id: "pdf",
+            title: "PDF"
+        }
+    });
+
+    var allScales = [0.5, 1, 1.5, 2];
+
+    var LayerExportAsset = React.createClass({
+
+        mixins: [FluxMixin],
+
+        propTypes: {
+            index: React.PropTypes.number.isRequired,
+            layer: React.PropTypes.object.isRequired,
+            exportAsset: React.PropTypes.object.isRequired
+        },
+
+        /**
+         * Delete this asset
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleDeleteClick: function () {
+            var document = this.props.document,
+                layer = this.props.layer,
+                index = this.props.index;
+
+            this.getFlux().actions.export.deleteLayerExportAsset(document, layer, index);
+        },
+
+        /**
+         * Update this asset's scale
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateScale: function (scale) {
+            var scaleNum = Number.parseFloat(scale);
```

Comment:
changed to use mathUtil.  IIRC, the reason this function is receiving a string value is because that seemed to be the way datalist works ... or maybe because I don't know how to use it correctly ... at any rate I think this is fairly safe.  If `null` is parsed, then the action will resolve it to 1